build.gradle.kts

```diff
@@ -67,6 +79,16 @@ kotlin {
             }
         }
     }
+    val linuxX64 by sourceSets.creating {
```

Comment:
Creating this source set isn't necessary, since the target will create one already.

Reply:
The problem I had was referencing via `dependsOn`. I was having trouble where this project wouldn't work in a multi platform build. Without creating a nativeMain, and setting dependsOn, which requires a source set. The `presets` from konan I tried wasn't working. I realize it's duplicated code but couldn't find a work around.

```
    val linuxX64 by sourceSets.creating {
        dependsOn(nativeMain)
    }
...
            "linuxX64" -> compilations["main"].defaultSourceSet.dependsOn(linuxX64)
```

Reply:
Hmm, does auto-complete/intelli-sense work in the IDE as it is?
If it doesn't then `compilations["main"].defaultSourceSet.kotlin.srcDir("src/nativeMain/kotlin")` should make it work. Otherwise it doesn't really matter I guess.

Reply:
KTS results in a bunch of errors in IntelliJ for me. I've tried wiping my settings a number of times but haven't gotten it to work. Essentially no auto complete, or type hinting. Running Intellij Community Latest.